packages/react-instantsearch-hooks/src/__tests__/usePoweredBy.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { usePoweredBy } from '../usePoweredBy';
+
+describe('usePoweredBy', () => {
+  test('returns the connector render state', () => {
+    const { url } = usePoweredBy();
+
+    expect(url).toBe(
+      `https://www.algolia.com/?utm_source=react-instantsearch&utm_medium=website&utm_content=localhost&utm_campaign=poweredby`
+    );
+  });
+
+  test('does not set `utm_content` when `window.location` is not defined', () => {
+    const originalWindow = { ...window };
+    const windowSpy = jest.spyOn(global, 'window', 'get');
+    const { location, ...windowWithoutLocation } = originalWindow;
+    // @ts-expect-error
+    windowSpy.mockImplementation(() => windowWithoutLocation);
+
+    const { url } = usePoweredBy();
+
+    expect(url).toBe(
+      `https://www.algolia.com/?utm_source=react-instantsearch&utm_medium=website&utm_content=&utm_campaign=poweredby`
```

Comment:
```suggestion
      `https://www.algolia.com/?utm_source=react-instantsearch&utm_medium=website&utm_campaign=poweredby`
```

shouldn't it be this?

Reply:
They're identical so I think it's fine to leave it this way, to not increase the code complexity.

Reply:
Apparently no: https://github.com/algolia/react-instantsearch/blob/master/packages/react-instantsearch-core/src/connectors/__tests__/connectPoweredBy.node.js#L16

The use case [isn't tested in instantsearch.js](https://github.com/algolia/instantsearch.js/blob/master/src/connectors/powered-by/__tests__/connectPoweredBy-test.ts) but [that's the behavior](https://github.com/algolia/instantsearch.js/blob/master/src/connectors/powered-by/connectPoweredBy.ts#L55-L58).

I have no idea whether it makes a difference in Google Analytics, but we've never received complaints so far I believe so it's probably no biggie. If we can avoid extra code, it's always nice.